tools/cli/commands/release.js

```diff
@@ -150,12 +156,39 @@ export async function scriptRouter( argv ) {
 				  - You will now likely want to start a new release cycle like so:
 				      jetpack release ${ argv.project } new-cycle \n`.replace( /^\t+/gm, '' );
 			break;
-		case 'append':
-		case 'new-cycle':
+		case 'amend':
+			await checkBranchValid( argv );
+			argv.script = `vendor/bin/changelogger`;
```

Comment:
I happened to try using this on changelogger itself, and ran into the fact that changelogger doesn't have `vendor/bin/changelogger`. Instead of adding yet another special case for that here, I went ahead and made #21783 to make such special cases unnecessary.

I note this could still fail if no one has done a `composer install` in the target project, though. Not sure if that's worth worrying about.